chart/templates/controller.yaml

```diff
@@ -122,7 +122,7 @@ spec:
       - name: setup-wds-kubeconfig
         image: quay.io/kubestellar/kubectl:1.27.8
         imagePullPolicy: Always
-        command: [ "bin/sh", "-c", "sh /mnt/config/get-kubeconfig.sh {{.Values.wds_cp_name}} false | base64 -d > /mnt/shared/wds-kubeconfig"]
+        command: [ "bin/sh", "-c", "sh /mnt/config/get-kubeconfig.sh '{{.Values.wds_cp_name}}' false | base64 -d > /mnt/shared/wds-kubeconfig"]

```

Comment:
This is a safety measure, guarding against the possibility that a user could (maliciously or mistakenly --- but never validly) puts a space or other word-separating character in the `wds_cp_name`, right? But what if the user puts a `'` in `wds_cp_name`?

Reply:
Also, FYI, the YAML source could remove one level of quoting by using the YAML syntax for an array of strings rather than the JSON syntax:

```
  command:
    - bin/sh
    - "-c"
    - sh /mnt/config/get-kubeconfig.sh {{.Values.wds_cp_name |  js}} false | base64 -d > /mnt/shared/wds-kubeconfig
```

Reply:
Regarding safety, there is also this:

```shell
bash-3.2$ echo -n "ugh "
ugh bash-3.2$ 
bash-3.2$ echo "-n" "ugh "
ugh bash-3.2$ 
bash-3.2$ echo '-n' "ugh "
ugh bash-3.2$ 
bash-3.2$ 
```

Reply:
Here is an illustration of some safety issues: https://go.dev/play/p/w0HEU6S3cpO

Reply:
Using the `js` function: https://go.dev/play/p/OexqtzIRlHz

Reply:
As described in https://helm.sh/docs/chart_template_guide/functions_and_pipelines/ , Helm charts can also call functions from Sprig (including one named `quote`).